app/views/observer/_form_observations.html.erb

```diff
@@ -98,16 +98,16 @@
           }) if include_naming %>
   </div>
 </div>
-<!-- HAS_SPECIMENS -->
-<% if has_specimens %>
+<!-- has_specimen -->
+<% if has_specimen %>
```

Comment:
Why the switch from plural to singular?  Observations can have multiple specimens (e.g., split collections in different herbaria).